src/custom/components/swap/EthFlow/index.tsx

```diff
@@ -255,15 +257,15 @@ export function EthWethWrap({
   const handleSwap = useCallback(
     async (showSwapModal?: boolean) => {
       try {
-        swapCallback(!!showSwapModal)
+        !!showSwapModal ? openSwapConfirm() : closeSwapConfirm()
```

Comment:
Since you are not returning anything, the type doesn't matter. What about not using double negation?

Reply:
Fixed: https://github.com/cowprotocol/cowswap/pull/1049/commits/354bbdf906b8439cecbfbff625c361452aae44e2